specs/FamilySafety.md

```diff
@@ -0,0 +1,114 @@
+  /// functionality as the Edge browser. 
+  /// Family Safety is a set of features available on Windows for managing children Internet 
+  /// priviliges. Microsoft account may be linked to have a family relationship, with adults 
+  /// and children. Adults have certain managements options over the children in their family, 
+  /// where each option is applied per-child. Edge browser provide the following options: 
+  /// Activity reporting, Web filtering and SafeSearch. Please see 
+  /// https://aka.ms/EdgeFamilySafetyFeatureOverview for more details on each feature.
+  [propget] HRESULT IsFamilySafetyEnabled([out, retval] BOOL* value);
+  /// Sets the `IsFamilySafetyEnabled` property.
+  [propput] HRESULT IsFamilySaFetyEnabled([in] BOOL value);
+
+  /// `GetFamilySafetyAllowedUris` and `SetFamilySafetyAllowedUris` allow developers to get and set
+  /// the list of URIs that will be allowed by the Family Safety filter even when in Block All mode.
+  /// Family Safety provides web filtering control in two modes: Allow-all and Block-all. In 
+  /// Allow-all mode, only sites that are blocked by the parents will be blocked. In 
+  /// blocked-all mode, only allowed sites that are allowed by the parents are allowed. In this 
+  /// scenario, apps using WebView2 will have their content blocked if enabled Family Safety in WebView2.
+  /// Please see https://aka.ms/EdgeFamilySafetyContentFiltering for more details.
+  ///
+  /// The `uris` parameter value are list of uris that is a wildcard string matched aganist the 
+  /// navigation uri. This is a glob style wildcard string in which a `*` matches zero or more characters and 
+  /// `?` matches exactly one character.
+  /// These wildcard characters can be escaped using a backslash just before
+  /// the wildcard character in order to represent the literal `*` or `?`.
+  /// 
+  /// | URI Example | Navigation URI | Match | Notes |
+  /// | ---- | ---- | ---- | ---- |
+  /// | `https://example.com/a/b/c` | `https://exmaple.com/a/b/c` | Yes | Matches exact uri |
+  /// | `*` | `https://example.com/a/b/c` | Yes | A single * will match all URIs |
+  /// | `https://example.com/*` | `https://exmaple.com/a/b/c` | Yes | Matches everything in example.com |
+  /// \snippet AppWindow.cpp CoreWebView2FamilySafety
+  HRESULT GetFamilySafetyAllowedUris([out] UINT32* uriCounts, [out] LPWSTR** uris);
+  /// Set the family safety allowed URIs. See GetFamilySafetyAllowedUris for more information.
+  HRESULT SetFamilySafetyAllowedUris([in] UINT32 urisCount, [in] LPCWSTR* uris);
+}
+```
+
+```c# (but really MIDL3)
+namespace Microsoft.Web.WebView2.Core
+{
+    // ...
+    runtimeclass CoreWebView2Environment
+    {
+        // ...
+        
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2EnvironmentOptions3")]
+        {
+            // ICoreWebView2EnvironmentOptions3 members
+            Boolean IsFamilySafetyEnabled { get; set; };
```

Comment:
For both APIs we want to remove brand names and use generic term "Parental Controls":

IsFamilySafetyEnabled becomes:
* AreParentControlsEnabled

FamilySafetyAllowedUris becomes:
* ParentalControlsAdditionalAllowedUris

But continue to use Family Safety in the documentation.